src/main/java/org/project36/qualopt/web/rest/ParticipantResource.java

```diff
@@ -108,8 +108,8 @@ public ParticipantResource(ParticipantRepository participantRepository) {
     @Timed
     public ResponseEntity<Participant> getParticipant(@PathVariable Long id) {
         log.debug("REST request to get Participant : {}", id);
-        Participant participant = participantRepository.findOne(id);
-        return ResponseUtil.wrapOrNotFound(Optional.ofNullable(participant));
+        Optional<Participant> participant = participantRepository.findById(id);
```

Comment:
findOne() can no longer pull by ID, the unwrapping of the Optional seems to be handled by ResponseUtil, so there should be no practical differences. If there is an issue, then it should be resolved via the Optional api rather than through a Null, as Java 8 and 9 is trying to move away from null checking I think.